backend/core/src/listings/listings-csv-exporter.service.ts

```diff
@@ -122,9 +122,9 @@ export class ListingsCsvExporterService {
         "Digital Application": formatYesNo(listing.digitalApplication),
         "Digital Application URL": listing.applicationMethods[1]?.externalReference,
         "Paper Application": formatYesNo(listing.paperApplication),
-        "Paper Application URL": cloudinaryPdfFromId(
-          listing.applicationMethods[0]?.paperApplications[0]?.file?.fileId
-        ),
+        "Paper Application URL": listing.applicationMethods[0]?.paperApplications
```

Comment:
what are your thoughts on pulling this out into a helper function?

Reply:
Sounds good! Updated 🥝